salt/_modules/metalk8s_cordon.py

```diff
@@ -0,0 +1,117 @@
+import logging
+
+from salt.exceptions import CommandExecutionError
+
+import metalk8s_kubernetes
```

Comment:
Weird, you "by pass" the "virtual check" + "virtual name" by importing directly the python module.

Would suggest to use `__salt__['metalk8s_kubernetes.<function>']` instead of importing the python module

Or if we have some "private" python function we want to use in several modules then we should have some `utils`
https://docs.saltstack.com/en/latest/topics/utils/index.html

Reply:
As we saw, `_utils` is not adapted to grabbing config items. Did that.